infra/cifuzz/fuzz_target.py

```diff
@@ -121,6 +124,32 @@ def is_reproducible(self, test_case):
         return True
     return False
 
+  def is_crash_valid(self, test_case):
+    """Checks if a crash was introduced by the pull request.
+
+    NOTE: If no old_build_target is specified the crash is assumed introduced
+    by the pull request.
+
+    Args:
+      test_case: The path to the test_case that triggered the crash.
+
+    Returns:
+      True if the crash was introduced by the current pull request.
+    """
+    if not self.old_build_path:
+      return True
+    exists_in_pr = self.is_reproducible(test_case,
+                                        os.path.dirname(self.target_path))
+    if not exists_in_pr:
+      logging.info('Crash is not reproducible.')
+      return False
+    exists_in_master = self.is_reproducible(test_case, self.old_build_path)
```

Comment:
let's use "ossfuzz" instead of "master", because OSS-Fuzz build might be up to 24hr old and not precisely follow the `master` branch at the moment

Reply:
Fixed.